runtime/compiler/z/codegen/J9UnresolvedDataReadOnlySnippet.cpp

```diff
@@ -67,12 +67,14 @@ J9::Z::UnresolvedDataReadOnlySnippet::UnresolvedDataReadOnlySnippet(
       intptr_t resolveDataAddress,
       TR::LabelSymbol *startResolveSequenceLabel,
       TR::LabelSymbol *volatileFenceLabel,
+      TR::LabelSymbol *snippetCallNextInstrLabel,
```

Comment:
How about calling it what is is, i.e. `returnLabel`?

Reply:
I chose this name it represents the nextInstruction after snippet call for GC, and in none of the path we actually return to this point.